base/docs/Docs.jl

```diff
@@ -287,7 +287,7 @@ isexpr(x, ts...) = any(T->isa(T, Type) && isa(x, T), ts)
 
 function unblock(ex)
     isexpr(ex, :block) || return ex
-    exs = filter(ex->!isexpr(ex, :line), ex.args)
+    exs = filter(ex -> !(isa(ex, LineNumberNode) || isexpr(ex, :line)), ex.args)
```

Comment:
yes, in macro expansions we call `scm_to_julia_` with `expronly=1`, so `:line` (and all other tree nodes) get translated to Exprs.